CONTRIBUTING.md

```diff
@@ -50,6 +52,17 @@ There are several linter commands available to help ensure the plugin follows th
 
 To test a WordPress plugin, you need to have WordPress itself installed. If you already have a WordPress environment setup, use the above Create Block Theme build as a standard WordPress plugin by putting the `create-block-theme` directory in your wp-content/plugins/ directory.
 
+### Repository Management
+
+Members of the [Block Themers GitHub team](https://github.com/orgs/WordPress/teams/block-themers) have write access to the repository. The team is made up of contributors who have:
```

Comment:
Do you think this would be a good place to mention that the team can also be found in #core-editor in the Make Slack?

Reply:
Yeah that's a good idea - I've added a Discussions section which includes links to #core-editor, #outreach (as CBT is often discussed there) and I moved the link to the GH project board there too in https://github.com/WordPress/create-block-theme/pull/614/commits/5dbf9115d9654fbb2f8c7cb814272ddbc8b71fb9.